### ISSUE TITLE
secret folder not getting created in C:\ProgramData\Docker location

### ISSUE THREAD

**Issue by monubamb** [dockersamples/newsletter-signup#2]
When I run the docker secret create signup-db-sa.password .\secrets\signup-db-sa.password command
I dont see the secrets folder created inside C:\ProgramData\Docker 

What am I missing. I did init a docker swarm on my Win 10 machine.

**Comment by sixeyed** [comment]
Hi, can you run `docker version` and confirm you're on 17.06? Also you should be looking for the secret file _inside_ the container, on the host it's stored in the swarm raft store.

**Comment by monubamb** [comment]
Thanks. That did work, My swarm was not working locally so had to jump to a VM. 
One other question.. I was trying to get volume mounting to work from with stack compose but it gives me the following error.. 
**This is what I am trying to do..**
  ports:
    - mode: host
      published: 80
      target: 80
    volume:
      - c:/shared:c:/shared
 **Error -** 
"volume Additional property volume is not allowed"

**Comment by sixeyed** [comment]
The property name is `volumes` - try this:

**Comment by sixeyed** [comment]
```
volumes:
  - C:\shared:C:\shared
```

**Comment by monubamb** [comment]
Thanks. That worked. I am at the final stage. Got my services up and running but have few outstanding issues..

Issue1 - My Web app cannot be accessed over Swarm IP (which in my case is 172.17.4.4 - says page cannot be displayed. So I had to do docker network inspect to get the NAT IP and access it that way..

Issue 2 - More critical... I am not able to access the DB using NAT IP as well from SSMS. Also dont see Prospects table created.. So somehow the web is not able to talk to DB even though I did find secrets folder with password on C:/programdata/docker/secrets on DB container..
I see this UI but data is not populated as far as I can tell..
![image](https://user-images.githubusercontent.com/333392/28086176-fb05c062-6632-11e7-88f6-39e2819e55e5.png)

Is this the correct password for DB... or is it DockerCon!!! base64 encoded
![image](https://user-images.githubusercontent.com/333392/28086205-1b4638c0-6633-11e7-9b75-
9144fda87039.png)

![image](https://user-images.githubusercontent.com/333392/28086299-5be987f6-6633-11e7-96a6-e42271f36cef.png)
